pyproject.toml

```diff
@@ -29,6 +29,7 @@ cryptography = "40.0.2"
 executing = "1.2.0"
 pydantic = "< 2"
 ipywidgets = "8.1.2"
+minio = "^7.2.7"
```

Comment:
Is it really necessary? Can it be made "optional", that is not in the main / required dependencies?

Reply:
Yes, I missed to remove it from env packages, it is only needed in rayjob's mnist_requirements file to make it available for a rayjob to download datasets.

Reply:
That means it can be removed from here right?

Reply:
Actually, We need a s3 compatible tool python package for interacting with storage bucket to download the datasets.. need to use [minio](https://min.io/docs/minio/linux/developers/python/minio-py.html) or [boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html) for the same
So we can't remove it ..

Reply:
It's only needed for testing, right? If yes can it be removed from the main dependencies and added to the testing ones?